build.gradle.kts

```diff
@@ -219,6 +219,7 @@ tasks.shadowJar {
     val shadeFilePrefix = shadePrefix.replace(".", "_")
     relocate("io.netty", "${shadePrefix}io.netty")
     relocate("META-INF/native/libnetty", "META-INF/native/lib${shadeFilePrefix}netty")
+    exclude("META-INF/com.google.dagger_dagger.version")
```

Comment:
minor: I would move it down under `relocate("dagger", ...` because now this breaks the semantic grouping of netty.

Reply:
Good one!